bin/node/runtime/src/lib.rs

```diff
@@ -1671,6 +1704,9 @@ construct_runtime!(
 		Uniques: pallet_uniques,
 		TransactionStorage: pallet_transaction_storage,
 		VoterList: pallet_bags_list::<Instance1>,
+		TargetList: pallet_bags_list::<Instance2>,
```

Comment:
If these two instances are only going to be used by the `stake-tracker`, they could be better named things like `StakeTrackerVoter` and `StakeTrackerTargets`.

Reply:
VoterList is used in the original Staking. TargetList, once tested, will be passed to Staking as well. It kind of felt strange to be renaming this back and forth, as those lists shouldn't care who tracks them and neither should the Staking pallet. 

Unfortunately I had to add an ugly VoterListTracker bags-list, but that will be dropped as soon as we have concluded that the original VoterList tracked by Staking and the new one are one and the same. 

Let me know if this makes sense.